Technical-Steering-Committee/projects.md

```diff
@@ -0,0 +1,180 @@
+
+#### Timeline
+
+As long as necessary to move the project to at least the Generally Available stage. The project shouldn't stay in this phase longer than a year. If this is the case, it's possible to return the project to Under Development.  
+
+#### Support
+
+The project is ready for the test and validation environments. The support is still provided on the best effort basis for the last version in which the technical preview was available. The project still can be moved back into Under Development or directly Archived.
+
+#### Recommended Use
+
+Put the project in testing and validation environments and provide feedback to Zowe to improve the project. 
+
+#### Possible transitions
+
+- From Technical Preview to Generally Available means that the project received the needed feedback and is ready to be put into production
+- From Technical Preview to Long Term Support means that the project is stable and mature. It belongs to the production
+- From Technical Preview to Archived means that after the feedback from the customers, the project found that it's not a good fit and the development won't continue
+
+### Generally Available
+
+#### Purpose
+
+To gather more feedback and improve the project before stabilizing it for Long Term Support. The project is stable, and we don't expect major changes, but unlike for Long Term Support, the changes may be needed. We believe that it's ok for GA products to become part of the production environments. The project may need to release major versions with breaking changes more often than the projects in Long Term Support. There should be no breaking changes in the minor releases.
+
+#### Timeline
+
+Supported for at least six months with updates. It may become LTS during this time, and the LTS rules will apply. 
+
+#### Support
+
+The community will support GA projects for at least six months after the initial release. Afterward, with the next major Zowe release version, the latest, the project should also become LTS or Archived if it didn't succeed.
+
+#### Recommended Use
+
+Put the project in the test, validation, and production environments. Provide feedback to the Zowe to improve the project. 
+
+#### Possible transitions
+
+- From Generally Available to Technical Preview means that we found out, based on the customer feedback, that the project needs a major rework for a longer life 
+- From Generally Available to Long Term Support means that the project is stable and mature. It belongs to the production 
+- From Generally Available to Archived means that the project wasn't a good fit and, as such, won't be further developed
+
+### Long Term Support
+
+#### Purpose
+
+The most stable version doesn't necessarily contain the latest and newest functionality, but it's ready to be deployed to the production environments. The Long Term Support is the most stable and longest supported version of any project. There are no breaking changes allowed within the major release. 
```

Comment:
> There are no breaking changes allowed within the major release.

Are major releases possible with LTS? That is, can something go from LTS v2 to LTS v3 directly? Or would that be GA v3 first and then LTS v3. If it's the latter, can we just say "There are no breaking changes allowed." ?

Reply:
Yes it's possible to go from LTS to LTS. 
If project doesn't feel the need to have a GA lifecycle stage it's ok.